sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -1920,7 +1921,8 @@ describe("Atom management - Authentication", function(): void {
             secondaryKey: "UreXLPWiP6Murmsq2HYiIXs23qAvWa36ZOL3gb9rXLs="
           }
         ],
-        enablePartitioning: true
+        enablePartitioning: true,
+        enableExpress: false
```

Comment:
There are a few tests that would fail due to the missing "enableExpress" property.
You have updated the `updateQueue` tests fine, please check `createTopic`, `createQueue`, and `updateTopic` tests.

Reply:
@HarshaNalluru I have committed changes. Please verify